cmd/list_components.go

```
@@ -58,5 +60,50 @@
 	}
 
 	output, err := l.FilterAndListComponents(stackFlag, stacksMap)
-	return output, err
+	if err != nil {
+		return nil, err
+	}
+
+	selectorFlag, _ := flags.GetString("selector")
+	if selectorFlag != "" {
```

golangci-lint (1.64.8)

[warning] `if selectorFlag != ""` has complex nested blocks (complexity: 9)

(nestif)